08-plot-ggplot2.Rmd

```diff
@@ -292,7 +292,7 @@ code to modify!
 > In the previous examples and challenge we've used the `aes` function to tell
 > the scatterplot **geom** about the **x** and **y** locations of each point.
 > Another *aesthetic* property we can modify is the point *color*. Modify the
-> code from the previous challenge to **color** the points by the "continent"
```

Comment:
I was following the standard convention in R documentation and the Journal of Statistical Softaware that functions are typeset in `monospace` and R packages are in **bold**.